ansible/roles/test/tasks/pfc_wd/config_test/config_test.yml

```diff
@@ -118,6 +118,7 @@
     - name: Config tests - Check forward action configuration.
       vars:
         command_to_run: "sonic-cfggen -j {{ run_dir }}/pfc_wd_fwd_action.json --write-to-db"
+        test_ignore_file: config_test_ignore_messages
```

Comment:
Can we use set fact to set global variable of test_ignore_file as it will be used across the test, including those wrong config tests to be expected having errors.

Reply:
we need this only for particular tests cases
setting global variable can mask some potential errors

Reply:
Can you help me understand what cases don't need this within config_test.yml?

Reply:
the exceptions I've added to config_test_ignore_messages are basically related to counters discovery errors. When syncd(orchagent?) is trying to find out which counters are available it queries all one by one and SAI reports whether particular counter is available. Unfortunately when counter is not supported/implemented/etc SAI also prints an error to the log and LogAnalyzer treats this as test failure.
 It is possible that SAI implementation on other platforms does not print such messages (or maybe all counters are available/implemented?)